authors/apps/articles/tests/test_articles.py

```diff
@@ -431,3 +432,14 @@ def test_email_article_link(self):
         em_response = self.client.get(
             reverse("article:detail", args=[slug]),  format='json')
         self.assertEqual(em_response.status_code, status.HTTP_200_OK)
+
+    def test_read_article(self):
+        """Tests that user is added to database after reading an article"""
+        self.client.credentials(
+            HTTP_AUTHORIZATION='Bearer ' + self.get_user_token())
+        response = self.client.get(
+            self.url + 'coding/', format='json')
+        user = User.objects.get(email="f.faraqhan91234@gmail.com")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertTrue(ReadingStat.objects.all().filter(
+            user=user.username, articles='coding'))
```

Comment:
Why are hard coding articles='coding'. Since your using fixtures you get that particular article using Article.objects.get(title='coding')